evap/middleware.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.views import redirect_to_login
+
+
+class RequireLoginMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        return self.get_response(request)
```

Comment:
Kind of sad that django doesn't give us a base class that already implements these, but well...